propertyestimator/protocols/simulation.py

```diff
@@ -10,12 +10,14 @@
 
 import numpy as np
 import yaml
-from simtk import unit, openmm
+from simtk import openmm, unit as simtk_unit
 from simtk.openmm import app
 
+from propertyestimator import unit
 from propertyestimator.thermodynamics import ThermodynamicState, Ensemble
 from propertyestimator.utils.exceptions import PropertyEstimatorException
-from propertyestimator.utils.openmm import setup_platform_with_resources
+from propertyestimator.utils.openmm import setup_platform_with_resources, openmm_quantity_to_pint,\
```

Comment:
I think it's becoming preferred to do multi-line imports using parentheses: https://www.python.org/dev/peps/pep-0328/

Reply:
Agree - will keep this for now as it's consistent, and open a PR to fix this across the codebase in the future